crates/data_components/src/spark_connect.rs

```diff
@@ -127,8 +127,8 @@ fn arrow_field_datatype_from_spark_connect_field_datatype(
                 Some(data_type) => {
                     let arrow_inner_type =
                         arrow_field_datatype_from_spark_connect_field_datatype(Some(*data_type))?;
-                    // Very smelly
-                    let field = Field::new("", arrow_inner_type, false);
+                    // The default name for the field is element
```

Comment:
I don't think this is a general purpose solution, see https://discord.com/channels/885562378132000778/1166447479609376850/1237526038700101644

Reply:
> Have any of you used DataFusion in databricks reading from an existing delta lake table that was written using pyspark? I'm trying to do so and getting annoying errors about the schemas not matching. ArrowException: Invalid argument error: column types must match schema types, expected List(Field { name: "item", data_type: Utf8, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }) but found List(Field { name: "element", data_type: Utf8, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }) at column index 10. It looks to me like there's a difference in how the data are stored. It's a list of strings and the "name" of the field is "item" vs "element".

Reply:
it's actually from spark-connect-rs, which seems to be always elements from what I can tell. But realise this is not needed at all.